2020-08-14 11:37:33 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 11:37:35 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 11:37:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 11:47:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 11:57:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 12:07:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 12:17:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 12:27:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 12:37:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 12:47:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 12:57:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 13:07:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 13:17:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 13:27:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 13:37:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 13:46:19 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 13:46:21 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 13:46:21 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 13:46:35 [WARN ] com.netflix.eureka.registry.AbstractInstanceRegistry.renew[360] - DS: Registry: lease doesn't exist, registering resource: CMSC-USC - 192.168.2.186:cmsc-usc:9000:1.0.0-SNAPSHOT
2020-08-14 13:46:35 [WARN ] com.netflix.eureka.resources.InstanceResource.renewLease[116] - Not Found (Renew): CMSC-USC - 192.168.2.186:cmsc-usc:9000:1.0.0-SNAPSHOT
2020-08-14 13:51:25 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 13:51:27 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 13:51:28 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 13:51:35 [WARN ] com.netflix.eureka.registry.AbstractInstanceRegistry.renew[360] - DS: Registry: lease doesn't exist, registering resource: CMSC-USC - 192.168.2.186:cmsc-usc:9000:1.0.0-SNAPSHOT
2020-08-14 13:51:35 [WARN ] com.netflix.eureka.resources.InstanceResource.renewLease[116] - Not Found (Renew): CMSC-USC - 192.168.2.186:cmsc-usc:9000:1.0.0-SNAPSHOT
2020-08-14 13:53:21 [WARN ] com.netflix.discovery.TimedSupervisorTask.run[73] - task supervisor timed out
java.util.concurrent.TimeoutException: null
	at java.util.concurrent.FutureTask.get(FutureTask.java:205) ~[?:1.8.0_101]
	at com.netflix.discovery.TimedSupervisorTask.run(TimedSupervisorTask.java:68) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 13:53:24 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 13:54:16 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 13:54:27 [WARN ] com.netflix.discovery.DiscoveryClient$3.notify[1335] - Saw local status change event StatusChangeEvent [timestamp=1597384467280, current=DOWN, previous=UP]
2020-08-14 13:54:47 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 13:54:48 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 13:54:49 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 13:55:50 [WARN ] com.netflix.discovery.TimedSupervisorTask.run[73] - task supervisor timed out
java.util.concurrent.TimeoutException: null
	at java.util.concurrent.FutureTask.get(FutureTask.java:205) ~[?:1.8.0_101]
	at com.netflix.discovery.TimedSupervisorTask.run(TimedSupervisorTask.java:68) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 13:55:53 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 13:57:36 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 13:58:37 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:00:05 [WARN ] com.netflix.discovery.TimedSupervisorTask.run[73] - task supervisor timed out
java.util.concurrent.TimeoutException: null
	at java.util.concurrent.FutureTask.get(FutureTask.java:205) ~[?:1.8.0_101]
	at com.netflix.discovery.TimedSupervisorTask.run(TimedSupervisorTask.java:68) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 14:00:05 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:06:10 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:06:10 [WARN ] com.netflix.discovery.TimedSupervisorTask.run[73] - task supervisor timed out
java.util.concurrent.TimeoutException: null
	at java.util.concurrent.FutureTask.get(FutureTask.java:205) ~[?:1.8.0_101]
	at com.netflix.discovery.TimedSupervisorTask.run(TimedSupervisorTask.java:68) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 14:06:10 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 14:10:41 [WARN ] com.netflix.discovery.TimedSupervisorTask.run[73] - task supervisor timed out
java.util.concurrent.TimeoutException: null
	at java.util.concurrent.FutureTask.get(FutureTask.java:205) ~[?:1.8.0_101]
	at com.netflix.discovery.TimedSupervisorTask.run(TimedSupervisorTask.java:68) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 14:10:41 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:10:41 [WARN ] com.netflix.discovery.DiscoveryClient$3.notify[1335] - Saw local status change event StatusChangeEvent [timestamp=1597385441498, current=DOWN, previous=UP]
2020-08-14 14:11:34 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:11:35 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:12:03 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 14:22:39 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 14:22:59 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:23:11 [WARN ] com.netflix.discovery.TimedSupervisorTask.run[73] - task supervisor timed out
java.util.concurrent.TimeoutException: null
	at java.util.concurrent.FutureTask.get(FutureTask.java:205) ~[?:1.8.0_101]
	at com.netflix.discovery.TimedSupervisorTask.run(TimedSupervisorTask.java:68) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 14:25:01 [WARN ] com.netflix.discovery.DiscoveryClient$3.notify[1335] - Saw local status change event StatusChangeEvent [timestamp=1597386301107, current=DOWN, previous=UP]
2020-08-14 14:29:21 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:29:22 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:29:23 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 14:38:17 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:38:18 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:38:22 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 14:39:06 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:39:14 [WARN ] com.netflix.discovery.DiscoveryClient.register[859] - DiscoveryClient_COM.CENTIFY.STORE.STORE-EUREKA/admin-tl:com.centify.store.store-eureka:9001 - registration failed Cannot execute request on any known server
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:857) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 14:39:14 [WARN ] com.netflix.discovery.InstanceInfoReplicator.run[125] - There was a problem with the instance info replicator
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:857) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 14:39:53 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:41:25 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:42:08 [WARN ] com.netflix.discovery.DiscoveryClient$3.notify[1335] - Saw local status change event StatusChangeEvent [timestamp=1597387328179, current=DOWN, previous=UP]
2020-08-14 14:43:33 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:43:34 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:43:35 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 14:44:15 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:45:21 [WARN ] com.netflix.discovery.TimedSupervisorTask.run[73] - task supervisor timed out
java.util.concurrent.TimeoutException: null
	at java.util.concurrent.FutureTask.get(FutureTask.java:205) ~[?:1.8.0_101]
	at com.netflix.discovery.TimedSupervisorTask.run(TimedSupervisorTask.java:68) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 14:45:46 [WARN ] com.netflix.discovery.DiscoveryClient$3.notify[1335] - Saw local status change event StatusChangeEvent [timestamp=1597387546531, current=DOWN, previous=UP]
2020-08-14 14:52:22 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:52:23 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:52:26 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 14:55:32 [WARN ] com.netflix.discovery.TimedSupervisorTask.run[73] - task supervisor timed out
java.util.concurrent.TimeoutException: null
	at java.util.concurrent.FutureTask.get(FutureTask.java:205) ~[?:1.8.0_101]
	at com.netflix.discovery.TimedSupervisorTask.run(TimedSupervisorTask.java:68) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 14:55:33 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:56:11 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:57:00 [WARN ] com.netflix.discovery.DiscoveryClient$3.notify[1335] - Saw local status change event StatusChangeEvent [timestamp=1597388220547, current=DOWN, previous=UP]
2020-08-14 14:58:13 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:58:14 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 14:58:15 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 14:58:25 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 14:58:25 [WARN ] com.netflix.discovery.DiscoveryClient.register[859] - DiscoveryClient_COM.CENTIFY.STORE.STORE-EUREKA/admin-tl:com.centify.store.store-eureka:9001 - registration failed Cannot execute request on any known server
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:857) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 14:58:25 [WARN ] com.netflix.discovery.InstanceInfoReplicator.run[125] - There was a problem with the instance info replicator
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:857) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 15:00:04 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 15:00:04 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 15:00:09 [WARN ] com.netflix.discovery.TimedSupervisorTask.run[73] - task supervisor timed out
java.util.concurrent.TimeoutException: null
	at java.util.concurrent.FutureTask.get(FutureTask.java:205) ~[?:1.8.0_101]
	at com.netflix.discovery.TimedSupervisorTask.run(TimedSupervisorTask.java:68) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 15:00:56 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 15:00:56 [WARN ] com.netflix.discovery.DiscoveryClient.register[859] - DiscoveryClient_COM.CENTIFY.STORE.STORE-EUREKA/admin-tl:com.centify.store.store-eureka:9001 - registration failed Cannot execute request on any known server
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:857) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 15:00:56 [WARN ] com.netflix.discovery.InstanceInfoReplicator.run[125] - There was a problem with the instance info replicator
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:857) ~[eureka-client-1.9.21.jar:1.9.21]
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121) ~[eureka-client-1.9.21.jar:1.9.21]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_101]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_101]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_101]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_101]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
2020-08-14 15:00:56 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 15:01:49 [WARN ] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute[130] - Request execution failed with message: java.net.SocketTimeoutException: Read timed out
2020-08-14 15:01:49 [WARN ] com.netflix.discovery.DiscoveryClient$3.notify[1335] - Saw local status change event StatusChangeEvent [timestamp=1597388509573, current=DOWN, previous=UP]
2020-08-14 15:02:04 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 15:10:17 [WARN ] com.netflix.config.sources.URLConfigurationSource.<init>[121] - No URLs will be polled as dynamic configuration sources.
2020-08-14 15:10:18 [WARN ] com.netflix.eureka.cluster.PeerEurekaNodes.updatePeerEurekaNodes[155] - The replica size seems to be empty. Check the route 53 DNS Registry
2020-08-14 15:14:08 [WARN ] com.netflix.discovery.DiscoveryClient$3.notify[1335] - Saw local status change event StatusChangeEvent [timestamp=1597389248449, current=DOWN, previous=UP]
